src/index.js

```diff
@@ -1,11 +1,15 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-import './index.css';
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+import "./index.css";
+import { ThemeProvider } from "@material-ui/core/styles";
```

Comment:
external libraries should be imported first, so move it below the react-dom import